src/style.css

```diff
@@ -0,0 +1,264 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+body {
+  background-color: rgb(243, 239, 239);
+  font-family: 'Comic Neue', cursive;
+}
+
+nav {
+  background-color: rgb(7, 7, 43);
+  display: flex;
+  justify-content: space-between;
+  padding: 0 1em;
+}
+
+.nav-logo {
+  background: url(./images/food-logo.png);
+  width: 4rem;
+  height: 4rem;
+  margin-left: 1.5rem;
+  background-size: cover;
+}
+
+.header_logo_container {
+  display: flex;
+  color: white;
+  font-weight: 700;
+  font-size: 25px;
+  align-items: center;
+}
+
+nav li {
+  color: white;
+  font-size: large;
+}
+
+a {
+  text-decoration: none;
+  color: white;
+}
+
+.nav_container {
+  display: flex;
+  flex-direction: row;
+  list-style: none;
+  padding: 1em 2em;
+  font-size: 20px;
+  font-weight: 700;
+  gap: 20px;
+  justify-content: center;
+}
+
+.nav_items_links:hover {
+  color: rgb(9, 236, 66);
+}
+
+.main_items_container {
+  display: grid;
+  grid-template-columns: 1fr 1fr 1fr;
+  grid-template-rows: 1fr 1fr 1fr;
+  width: 90%;
+  margin: 3em auto;
+  gap: 20px;
+  max-width: 800px;
+}
+
+.item_contianer {
+  width: 100%;
+  margin-bottom: 1em;
+  display: flex;
+  flex-direction: column;
+}
+
+.item_contianer img {
+  width: 100%;
+  margin-bottom: 1em;
+}
+
+.name_like_conatiner {
+  display: flex;
+  justify-content: space-between;
+  margin-bottom: 1em;
+  font-size: 20px;
+  font-weight: 700;
+  width: 70%;
+  align-self: center;
+}
```

Comment:
- [ ] OPTIONAL
To improve the **user experience** it will be good practice to ensure that the _Likes_ buttons, and similar other buttons are behaving as buttons when the mouse hovers on them. We can improve upon this aspect by adding the CSS attribute `cursor: pointer` to the _Comment_ button on _mouse hover_ events.